source/sync/production-checklist.txt

```diff
@@ -0,0 +1,66 @@
+.. _realm-sync-best-practices:
+.. _sync-production-checklist:
+
+================================
+Device Sync Production Checklist
+================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Best Practices
+--------------
+
+In addition to the information in this guide, we recommend following these best
```

Comment:
Nit: this opening sentence for the page doesn't make sense where this is the only content on the page. 
```suggestion
We recommend following these best
```